docs/dev/table/python/common_questions.md

```diff
@@ -33,10 +33,11 @@ You can download a [convenience script]({{ site.baseurl }}/downloads/setup-pyfli
 You can specify the version parameter to generate a Python virtual environment required for the corresponding PyFlink version, otherwise the most recent version will be installed.
 
 {% highlight bash %}
+$ curl -O {{ site.baseurl }}/downloads/setup-pyflink-virtual-env.sh
```

Comment:
Not sure why {{ site.baseurl }} is missing in the HTML file. @sjwiesman Do you know how to generate the URL automatically? Thanks.